File: src/main/java/br/com/leonardoferreira/jirareport/exception/HolidaysAlreadyImported.java
package br.com.leonardoferreira.jirareport.exception;

import lombok.NoArgsConstructor;
import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@NoArgsConstructor
@ResponseStatus(HttpStatus.BAD_REQUEST)
public class HolidaysAlreadyImported extends RuntimeException {
    private static final long serialVersionUID = 1189381143800680941L;

    public HolidaysAlreadyImported(final Exception e) {
        super(e);
    }

}

File: common-library/src/main/kotlin/br/com/jiratorio/exception/HolidaysAlreadyImported.kt
package br.com.jiratorio.exception

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

@ResponseStatus(value = HttpStatus.BAD_REQUEST)
class HolidaysAlreadyImported(
    message: String? = null,
    cause: Exception? = null
) : RuntimeException(message, cause)
